styles.css

```diff
@@ -313,6 +330,23 @@ button:active {
   justify-content: space-around;
 }
 
+.list-boxPop > ul > li {
+  font-size: 12px;
+  line-height: 16px;
+  height: 36px;
+  padding-right: 7px;
+  padding-left: 7px;
+  padding-top: 7px;
+  padding-bottom: 7px;
+  margin-right: 5px;
+  margin-bottom: 5px;
+  font-weight: 600;
+  color: #3a4a42;
+  background: #ebf0ee;
+  border-radius: 4px;
```

Comment:
- [ ] As suggest by the previous reviewer, please stick to the stick to the Figma design as much as possible. Therefore I suggest you reduce the `height` of the project technologies to follow the Figma design.
### Your design :point_down: 
![Screenshot from 2023-06-21 19-33-50](https://github.com/yusufyusie/yusufyusie.github.io/assets/91142432/4c9e5d14-8776-45d3-8bfb-ee5297b1d668)
### What it's supposed to look like :point_down: 
![Screenshot from 2023-06-21 19-42-03](https://github.com/yusufyusie/yusufyusie.github.io/assets/91142432/fc026538-ddf3-44f8-8dd2-b2d16c4bf229)